README.md

```diff
@@ -1,7 +1,25 @@
 # Helminator
 
-## OS Variables
+## TL;DR
 
-`HELMINATOR_ROOT_DIR` -> directory to scan  
-`HELMINATOR_SLACK_API_TOKEN` -> Slack API token  
-`HELMINATOR_SLACK_CHANNEL` -> Slack channel to send message to (#kubernetes)
+Helminator scans your Ansible roles for Helm repositories and versions.
+It then checks if there is an update to a given Helm chart is availible and sends out a Slack notification.
```

Comment:
`It then checks if there is an update to a given Helm chart availible and sends out a Slack notification.`